calm_adapter/calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/CalmQuery.scala

```diff
@@ -4,15 +4,40 @@ import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 
 sealed trait CalmQuery {
-  def key: String
+  def keys: List[String]
   def value: String
-  def queryExpression = s"$key=$value"
+  def logicalOperator = "OR"
+  def relationalOperator = "="
+  def queryExpression =
```

Comment:
Is this clearer than just having a list of `queryExpressions` on the `CalmQuery` being something like `List(s"Created=${formatDate(date)}")` or similar?

This also means you can only share the `logicalOperator` and value across fields. 

Saying this, ☝️ is exactly what we we're doing.

Reply:
I think this is fine, the options are either having the abstraction like we have now or just every case class defining the `queryExpression` directly. I think either methods work but want to get this up and running so going to merge as is.